conda_forge_tick/utils.py

```diff
@@ -1232,12 +1232,35 @@ def change_log_level(logger, new_level):
         logger.setLevel(saved_logger_level)
 
 
+@overload
```

Comment:
These changes are not needed anymore because at most one token is hidden

Reply:
Yes, this was a note for myself :D
code related to multi-token hiding removed in 29b75a07589ef0326a5ff250e70ae6d7f6599b8b